lunch-application/src/theme/EditTopFive.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="container">
+    <div class="editTopFive">
+    <h3>Edit Top Five</h3>
+    <table>
+      <tbody>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="firstChoice">First Choice</label>
+          </td>
+          <td>
+            <select id="firstChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="secondChoice">Second Choice</label>
+          </td>
+          <td>
+            <select id="secondChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="thirdChoice">Third Choice</label>
+          </td>
+          <td>
+            <select id="thirdChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="fourthChoice">Fourth Choice</label>
+          </td>
+          <td>
+            <select id="fourthChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <label id="choiceLabel" for="fifthChoice">Fifth Choice</label>
+          </td>
+          <td>
+            <select id="fifthChoice" class= "form-control">
+              <option v-for="(data, index) in restaurantData" :key="index">{{data.restaurantName}}</option>
+            </select>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <router-link to="/Home" tag="button" class="btn">Back</router-link>
+    <input class="btn btn-primary" type="submit" value="Submit" v-on:click="submitUpdate()" >
+    </div>
+  </div>
+</template>
+<script>
+  import axios from 'axios'
+  import VueOnToast from 'vue-on-toast'
+
+  export default {
+    data () {
+      return {
+        restaurantData: [],
+        topFiveData: [],
+        errors: []
+      }
+    },
+    created () {
+      axios.get('http://localhost:48146/lunchdata')
```

Comment:
"http://localhost:48146/"  -- you'll want to parameterize this at some point so that it could be deployed to an environment down the line. Take a look at EmployeeRewards for an example:

http://d03tfsapp01:8080/tfs/IT/InternalApps_Git/_git/EmployeeRewards?path=%2Fsrc%2Fcommon%2FemployeeRewardsService.js&version=GBmaster&_a=contents